parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -411,7 +425,22 @@ try
     }
 
     New-ExternalHnsNetwork -IsDualStackEnabled $global:IsDualStackEnabled
+
+    # Install calico using the kubeconfig at KubeDir/config, if enabled
+    if ($global:WindowsCalicoPackageURL) {

```

Comment:
we need to install calico after Check-APIServerConnectivity since it needs to access the api server

Reply:
could we also just move the Check-APIServerConnectivity call up here before the calico installation? I guess the main difference there would be we do the connectivity check before installing/starting the kubelet service

Reply:
Yes. I think that we can call Check-APIServerConnectivity and then install calico services